astacus/coordinator/plugins/clickhouse/object_storage.py

```diff
@@ -107,6 +96,29 @@ def get_storage(self) -> Iterator[BaseTransfer[Any]]:
             yield self._storage
 
 
+def copy_items_between(
```

Comment:
extracted method from class for testing purpose.
`test_steps` uses an abstraction `MemoryObjectStorage`, which does not reference `copy_items_between`.
Added a test in `test_copy_items_between_with_direct_copy_method` in order to verify `emit_direct_copy_progress` for `DirectCopyConfig`.

Reply:
IMHO not really important to have it private, especially since it's tested now; let's have it public